<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionPreferencesPage</class>
 <widget class="QWidget" name="ConnectionPreferencesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="listenInterfaceGroup">
     <property name="title">
      <string>listen_interfaces</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="listenInterfaces"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="networkAdapterLabel">
        <property name="text">
         <string>network_adapter</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="portLabel">
        <property name="text">
         <string>port</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="port">
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::UpDownArrows</enum>
        </property>
        <property name="minimum">
         <number>1024</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="encryptionGroup">
     <property name="title">
      <string>encryption</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="encryptIncoming">
        <property name="text">
         <string>require_for_incoming</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="encryptOutgoing">
        <property name="text">
         <string>require_for_outgoing</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="privacyGroup">
     <property name="title">
      <string>privacy</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="enableDht">
        <property name="text">
         <string>enable_dht</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enableLsd">
        <property name="text">
         <string>enable_lsd</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="enablePex">
        <property name="text">
         <string>enable_pex</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
